src/pages/home/report/ReportActionsView.js

```diff
@@ -352,7 +352,7 @@ class ReportActionsView extends React.Component {
 
     render() {
         // Comments have not loaded at all yet do nothing
-        if (!_.size(this.props.reportActions)) {
+        if (!_.size(this.props.reportActions) && !this.props.network.isOffline) {
```

Comment:
I think this needs a comment. It's not obvious why we should choose to return `null` when there are no report actions _and_ we are online.

Reply:
It also feels like maybe we should not be rendering the `ReportActionsView` at all if we are offline and have no actions.

Reply:
Going to relook into it because we now show the composer box while the chats are loading, so I think I can get rid of most of it.

Reply:
>I guess this whole flow is a bit messed up.

I agree and think it's a possible sign that we are gonna make it even more messed up than it already is. 

I would propose we stop exploring code changes for a moment and instead try to pull things back into the realm of problem/solution. 

**Next steps:**

1. Figure out what problem we need to solve (I think we have mostly done this on the issue)
2. Look at how things work today. Does the existing code easily support it or not? Does an obvious solution present itself or not? What solution should we ultimately go with?
3. Summarize our best idea and take it to a few people to see what they think (either `#engineering-chat` or `#expensify-open-source` seems good)
4. Agree on a plan forward + make sure everyone understands the general thought process behind the changes
5. Make the changes

Reply:
@aimane-chnaif No, @techievivek has been OOO - so you're not missing out on anything I believe 🙂

Reply:
Catching up, I will open a thread to discuss this issue in the open-source channel by tomorrow.

Reply:
I have opened a discussion [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1673246681648109) feel free to jump to add your thoughts.